woocommerce/woocommerce-gutenberg-products-block
# Show express payment methods in the Cart block

Fixes #2433.

This PR:
1. Adds express payment methods to the Cart block.
2. Makes the 'Proceed to Checkout' button in Cart to be sticky _only_ if it's below the viewport. Otherwise it gets positioned normally. That was discussed with @LevinMedia.

### Screenshots

_Partially sticky 'Proceed to Checkout' button:_
<img src="https://user-images.githubusercontent.com/3616980/90121409-3db19780-dd5c-11ea-93cb-e2a61dfac283.gif" alt="" width="374" />

_Express payment methods in Cart block:_
<img src="https://user-images.githubusercontent.com/3616980/90121004-b9f7ab00-dd5b-11ea-8272-9b63317765ad.jpeg" alt="" width="444" />

### How to test the changes in this Pull Request:

1. Go to the Cart block in a mobile or with a narrow window size and verify the 'Proceed to Checkout' button is no longer sticky once you pass its expected position.
2. If you have express payment methods set up, verify they appear in the Cart block and clicking on them triggers the payment process.

### Release notes

With the introduction of Express payment methods in the Cart block, we updated several CSS class names of the elements responsible to display them:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/main/docs/theming/class-names-update-330.md

### Changelog

> Express payment methods are now visible in the Cart block too and 'Proceed to Checkout' button stops being sticky after passing it's expected position on mobile.
